juriscraper/oral_args/united_states/federal_appellate/ca2.py

```diff
@@ -39,11 +57,66 @@ def _download(self, request_dict={}):
         r = self.request["session"].post(self.url, params=self.parameters)
         return html.fromstring(r.content)
 
-    def _process_html(self):
-        for row in self.html.xpath("//table[@border='1']"):
+    def _extract_cases_from_html(self, html):
+        for row in html.xpath("//table[@border='1']"):
             link = row.xpath(".//a")[0].get("href")
-            docket = row.xpath(".//a/nobr/text()")[0]
-            name, date = row.xpath(".//td/text()")
-            self.cases.append(
-                {"docket": docket, "url": link, "name": name, "date": date}
+            docket = (
+                row.xpath(".//a/nobr/text()")[0]
+                if row.xpath(".//a/nobr/text()")
+                else None
             )
```

Comment:
I think we can just avoid the two exceptions that occur in the backscrape since June.